pkg/config/remote/service/service_test.go

```diff
@@ -123,6 +123,7 @@ func TestServiceBackoffFailure(t *testing.T) {
 	uptaneClient := &mockUptane{}
 	clock := clock.NewMock()
 	service := newTestService(t, api, uptaneClient, clock)
+	defer service.Stop()
```

Comment:
You could add `t.Cleanup(func() { service.Stop() })` within `newTestService` to always guarantee it is cleaned up, and you wouldn't need to modify each test.

Reply:
Oh, TIL and I like this! 🙇🏻